lib/dlss/capistrano/tasks/check_ruby_version.rake

```diff
@@ -0,0 +1,39 @@
+namespace :ruby do
+  desc 'Retrieve ruby versions'
+  task :versions do
+    on roles(:all), in: :sequence do |host|
+      default_ruby = capture('rvm list default string').chomp.chomp.split('-').last
+      system_rubies = capture('rvm list rubies').split.grep(/ruby/).map { |version| version.delete_prefix('ruby-') }
+      info "#{host},#{default_ruby},#{system_rubies.join(',')}"
+    end
+  end
+
+  desc 'Report ruby versions'
+  task :check_version do
+    on roles(:all), in: :sequence do |host|
+      app_ruby = Bundler::LockfileParser.new(Bundler.read_file('Gemfile.lock')).ruby_version.split.last.split('p').first
+      default_ruby = capture('rvm list default string').chomp.chomp.split('-').last
+      system_rubies = capture('rvm list rubies').split.grep(/ruby/).map { |version| version.delete_prefix('ruby-') }
+      info "#{host} - App: #{app_ruby}, Default: #{default_ruby}, Installed: #{system_rubies.join(', ')}"
+    end
+  end
+
+  desc 'Verify ruby version'
+  task :verify_version do
+    on roles(:all), in: :sequence do |host|
+      app_ruby = Bundler::LockfileParser.new(Bundler.read_file('Gemfile.lock')).ruby_version.split.last.split('p').first
+      default_ruby = capture('rvm list default string').chomp.chomp.split('-').last
```

Comment:
what happens if the Gemfile.lock does not specify a ruby version at all?  seems like it would fail with a nil exception?

Reply:
Yes, at this point that's likely. This is a point of the discussion yesterday that will need final ironing out. Once we decide on the final approach we'll need a better guard here.

Reply:
Yes, that'd be a good one to add.

Reply:
agree we'd want a clear error if we're not finding something we're epecting.